tests/flow/declare_implements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,28 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test.js 1`] = `
+declare module "example" {
+  declare interface EventEmitter {
+    on(type: string, listener: Function): void;
+    off(type: string, listener: Function): void;
+  }
+
+  declare export class Terminal implements EventEmitter {
+    on(type: string, listener: Function): void;
+    off(type: string, listener: Function): void;
+  }
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+declare module "example" {
+  declare interface EventEmitter {
+    on(type: string, listener: Function): void;
+    off(type: string, listener: Function): void;
+  }
+
+  declare export class Terminal {
```

Comment:
The `implements EventEmitter` is missing.

Reply:
Looks like it's not exposed in the AST https://flow.org/try/#0CYUwxgNghgTiAEkoGdnwCohgWwJYDsoJ5dsAHCEbEfAFzQFEA3G2hvW2reAbwCgAvnyA

Reply:
@duailibe Should open an issue on flow about this if you haven't. (Or I can if you want.)

At a glance it looks like it's exactly the same problem as with [mixins](https://github.com/facebook/flow/issues/5463), so if anyone is up to making a PR to flow you could probably solve both without much work!

Reply:
Opened #5463, thanks!

Reply:
If the flow parser doesn’t parse it, does it mean that the type checker doesn’t use it? So it’s not a blocker

Reply:
Unfortunately not: the AST exposed to JS like prettier by `flow-parser` is translated from the AST used internally, lossily. The type checker has access to an AST which contains those nodes, I'm almost certain.

That said I don't think it should necessarily be a blocker anyway, since prettier has been doing the same with [`mixins`](https://github.com/prettier/prettier/pull/3391) for months.

Reply:
Yep, @bakkot is right. I don't think there's much we can do...

I opened https://github.com/facebook/flow/pull/6118 so we can hopefully land those soon